lib/screens/word/word_details.dart

```diff
@@ -34,59 +34,64 @@ class WordDetails extends StatelessWidget {
               ),
             ),
             backgroundColor: bgColor,
-            body: Column(
-              crossAxisAlignment: CrossAxisAlignment.center,
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                ArabicText(
-                  word.arabic,
-                  fontSize: 48,
-                ),
-                Row(
+            body: SingleChildScrollView(
+              child: SizedBox(
+                height: MediaQuery.of(context).size.height,
```

Comment:
@kmtusher97 Bhaia,
`MediaQuery.of(context).size.height` gives us the whole physical area of the screen. Some mobile devices have notches which overlap with Screen Widgets. 
![image](https://github.com/ashrafuzzaman/learn-quran/assets/33915686/07cd6f75-73ca-4164-853e-b117ed21db6f)


To avoid these issues we can either use `MediaQuery.of(context).size.height - MediaQuery.of(context).padding.top` or we can wrap the body of `Scaffold` with `SafeArea` widget ([SafeArea Docs](https://api.flutter.dev/flutter/widgets/SafeArea-class.html)).

Sorry, I forgot to mention it in our call.

Reply:
Done